src/main/java/com/Saurabh/GitHubAiAPI/StringOperations.java

```diff
@@ -18,13 +18,13 @@ public static int countVowels(String input) {
         return vowelCount;
     }
 
```

Comment:
There are multiple typos in the comment. It should be 'Method to replace all vowels in the input string with '*'.